packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-inline-checkout-button-strategy.ts

```diff
@@ -245,25 +258,14 @@ export default class PaypalCommerceInlineCheckoutButtonStrategy implements Check
         return shippingOptionToSelect;
     }
 
-    private async _resetCustomersAddress(): Promise<void> {
-        const emptyAddress = this._getEmptyAddress();
-
-        try {
-            await this._store.dispatch(this._billingAddressActionCreator.updateAddress(emptyAddress));
-            await this._store.dispatch(this._consignmentActionCreator.updateAddress(emptyAddress));
-        } catch (error) {
-            throw new Error(error);
-        }
-    }
-
-    private _transformAddress(address?: Partial<BillingAddressRequestBody>): BillingAddressRequestBody {
+    private _getAddress(address?: Partial<BillingAddressRequestBody>): BillingAddressRequestBody {
         return {
-            firstName: address?.firstName || 'Fake',
-            lastName: address?.lastName || 'Fake',
-            email: address?.email || 'fake@fake.fake',
+            firstName: address?.firstName || '',
+            lastName: address?.lastName || '',
+            email: address?.email || '',
             phone: '',
             company: '',
-            address1: address?.address1 || 'Fake street',
+            address1: address?.address1 || '',
             address2: '',
```

Comment:
Same can we set `address2` value if present?

Reply:
Is it's ok to use `address?.address1` for `address2`?

Reply:
Unfortunately, PayPal didn't pass the data for `address2`

Reply:
No think should use `address2` if provided? Is there a flag that we can pass to get that data 🤔 https://developer.paypal.com/docs/api/orders/v2/#definition-address_portable looking at this we should be able to get `address_line_2`, or is that the wrong doc ?

Reply:
Here is another doc for Accelerated Checkout feature, but there is no information about an order interface that we can get from `actions.order.get()`

https://developer.paypal.com/beta/accelerated-checkout/

Reply:
Discussed offline and seems the accelerated beta is only for US only. Should be ok in that case.